config/locales/en.yml

```diff
@@ -30,4 +30,8 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  enumerations:
```

Comment:
I believe it would be interesting to create an individual file for this entity. In the future we will do this with the others.

example:
config/locales/pt-BR/enums/medical_shift/workloads.yml

Reply:
It's interesting! I created an i18n for medical_shifts workload `config/locales/enums/medical_shifts/workloads.yml`. I let the locale with `en` language because is the default of the project. I think we can create a new issue to change locale to pt-BR